### PR TITLE
CB-8933 - CIS - Harden SSH Configurations

### PR DESCRIPTION
**Pull request by bormanbhaskar**
CB-8933:  CIS - Harden SSH Configurations.

### CODE REVIEW

**Review by risdenk — APPROVED** [review]

File: saltstack/base/salt/top.sls [hunk 1]
```diff
@@ -15,7 +15,6 @@ base:
     - fluent
 {% endif %}
     - ccm-client
-{% if salt['environ.get']('INCLUDE_CIS') == 'Yes' %}
```
Comment (keyki) [inline saltstack/base/salt/top.sls:18]: endif should be deleted as well.

Comment (bormanbhaskar) [reply saltstack/base/salt/top.sls:18]: Thanks, it's addressed now.

**Review by risdenk — APPROVED** [review]

**Review by keyki — APPROVED** [review]

**Comment by keyki** [comment]
It looks like after the merge the image burning jobs are failing. @pkedvessy can you please link the failing job or the error message?

**Comment by pkedvessy** [comment]
http://build.eng.hortonworks.com:8080/job/cloudbreak-multi-packer-image-builder-salt-v3/2052/
http://build.eng.hortonworks.com:8080/job/cloudbreak-multi-packer-image-builder-salt-v3/2053/
http://build.eng.hortonworks.com:8080/job/cloudbreak-multi-packer-image-builder-salt-v3/2054/

**Pull request merged**